src/lib/pages/library/ui/library-page.svelte

```diff
@@ -4,9 +4,9 @@
   import { EpisodeCard } from '$lib/widgets/episode';
   import { UserContainer } from '$lib/entities/user';
   import { SignInButton, SignOutButton } from '$lib/features/authenticate';
-  import { getEpisode } from '../lib/get-episode';
+  import { fetchEpisode } from '$lib/entities/episode/api/fetch-episode';
```

Comment:
issue: according to FSD limitations, we're not allowed to dig into the structure of another slice